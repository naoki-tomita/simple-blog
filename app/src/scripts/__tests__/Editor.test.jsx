import React from 'react';
import renderer from 'react-test-renderer';
import { Editor } from '../Editor';

describe("Editor", () => {
  it("should render successful.", () => {
    expect(renderer.create(
      <Editor />
    )).toMatchSnapshot();
  });
})
