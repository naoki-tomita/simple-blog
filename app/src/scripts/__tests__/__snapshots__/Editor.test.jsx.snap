// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Editor should render successful. 1`] = `
<div
  style={
    Object {
      "display": "none",
    }
  }
>
  <div
    className="dialog-container"
  >
    <div
      className="dialog-title"
    >
      記事を書く
    </div>
    <div
      className="dialog-content"
    >
      <div
        className="cp_iptxt"
      >
        <input
          className="ef"
          onChange={[Function]}
          placeholder=""
          type="text"
          value=""
        />
        <label>
          タイトル
        </label>
        <span
          className="focus_line"
        />
      </div>
      <div
        className="cp_iptxtarea"
      >
        <textarea
          className="ef"
          onChange={[Function]}
          rows={1}
          type="text"
          value=""
        />
        <label>
          本文
        </label>
        <span
          className="focus_line_t"
        />
      </div>
      <div
        className="dialog-footer"
      >
        <button
          className="btn-sticky"
          onClick={[Function]}
        >
          投稿する
        </button>
        <span
          style={
            Object {
              "display": "inline-block",
              "width": "12px",
            }
          }
        />
        <button
          className="btn-sticky"
        >
          閉じる
        </button>
      </div>
    </div>
  </div>
  <div
    className="dialog-backdrop"
  />
</div>
`;
